src/blueapi/core/context.py

```diff
@@ -106,7 +106,7 @@ def with_device_module(self, module: ModuleType) -> None:
     def with_dodal_module(self, module: ModuleType, **kwargs) -> None:
         from dodal.utils import make_all_devices
 
-        for device in make_all_devices(module, **kwargs).values():
+        for device in make_all_devices(module, **kwargs)[0].values():
             self.device(device)
```

Comment:
I think we want the issues logged

```suggestion
        devices, exceptions = make_all_devices(module, **kwargs)
        for device in devices.values():
            self.device(device)
        
        num_failed = len(exceptions)
        LOGGER.warning(f"{num_failed} devices failed to connect: {exceptions}")
```